src/main/java/de/tum/in/www1/artemis/service/exam/ExamService.java

```diff
@@ -1226,11 +1226,9 @@ public SearchResultPageDTO<Exam> getAllOnPageWithSize(final PageableSearchDTO<St
      * @param user     The user for whom to fetch all available exercises
      * @return exam page
      */
-    public Page<Exam> getAllActiveExams(Pageable pageable, final User user) {
-        final Page<Exam> examPage;
+    public Page<Exam> getAllActiveExams(final Pageable pageable, final User user) {
         // active exam means that exam has visible date in the past 7 days or next 7 days.
-        examPage = examRepository.findAllActiveExamsInCoursesWhereInstructor(user.getGroups(), pageable, ZonedDateTime.now().minusDays(7), ZonedDateTime.now().plusDays(7));
-        return examPage;
+        return examRepository.findAllActiveExamsInCoursesWhereInstructor(user.getGroups(), pageable, ZonedDateTime.now().minusDays(7), ZonedDateTime.now().plusDays(7));
```

Comment:
Improvement suggestion: Extract the magic number 7 into a static final constant.